grammars/arm-expression-string.tmLanguage.json

```diff
@@ -128,33 +129,37 @@
 			]
 		},
 		"functionname": {
-			"match": "(?x) \\s* (?: ({{ns-userfunc}}) | (parameters) | (variables) | ({{runtime}}) | ({{builtin}}) | ({{id}}) ) (?!{{idchar}} (?# Make sure we don't match a well-known name like 'add' inside something like 'add2'))",
+			"match": "(?x) \\s* (?: {{ns-userfunc}} | (parameters) | (variables) | ({{runtime}}) | ({{builtin}}) | ({{id}}) ) (?!{{idchar}} (?# Make sure we don't match a well-known name like 'add' inside something like 'add2'))",
 			"captures": {
 				"1": {
-					"$comment": "namespace + user function name",
-					"name": "{{scope-usernamespaceandfunction}}"
+					"$comment": "user namespace (capturing group inside ns-userfunc)",
+					"name": "{{scope-usernamespace}}"
 				},
 				"2": {
-					"$comment": "dot from capturing group inside ns-userfunc",
+					"$comment": "dot (capturing group inside ns-userfunc)",
 					"name": "{{scope-usernamespaceandfunction-dot}}"
```

Comment:
Admittedly I don't fully understand the colorization stuff - but elsewhere you removed `usernamespaceandfunction`. Is this one still supposed to be here?

Reply:
Yeah, that's the dot between the namespace and function. Thanks for asking.